### PR TITLE
Add content change audit log for session edits

### PR DESCRIPTION
**Pull request by hasparus**
## Summary

Adds an audit trail for session content edits, tracking all changes to core session fields and dynamic session-field answers. Both facilitator self-edits and organizer panel edits write to a shared `ContentChangeLog` table through one service, with a new activity-log view in the panel to review edit history.

## Key Changes

- **New `ContentChangeLog` model** (`adapters/db/django/models.py`): event, session, user, `changes` (JSON), timestamp. Migration `0082`.
- **`SessionContentEditService`** (`mills/chronology.py`): the single transactional boundary for a session content edit. `apply()` persists the session update, field values, **and facilitator assignment**, then writes the audit row — all in one `transaction.atomic()`.
  - `diff_session_content()`: pure function comparing old vs. new state (core columns + dynamic fields), returning an **identity-only** change list (`field` key / `field_id`, `old`, `new` — no display text).
  - Cover-image uploads logged as `(updated)`; empty/`None` normalized to avoid no-op diffs.
  - Write payload bundled into a `SessionContentEditData` DTO (`facilitator_ids=None` ⇒ leave untouched, used by the self-edit path).
- **`SessionSelfEditService`** delegates persistence to `SessionContentEditService`, so both edit paths produce identical audit rows.
- **Panel** (`gates/.../panel/views/proposals.py`): new read-only `ContentLogPageView` at `/panel/event/<slug>/proposals/log/`; `ProposalEditPageView` now calls `session_content_edit.apply(...)` (migrated off the direct `uow` session/field/facilitator writes).
- **Render-time i18n of labels**: nothing user-facing is stored in the log. Core-column labels are translated per request via a `content_field_label` template filter; dynamic session-field names resolve live from the event's current fields.
- **Repos**: `ContentChangeLogRepository`; shared `change_log_base.base_log_fields()` deduplicates DTO building across the schedule and content logs.
- **Template**: `content-log.html` uses `{% tessera_table %}` per the design system.
- **Tests**: unit tests for `diff_session_content` (core columns, cover image, session fields); integration tests for the log view (permissions, empty log, edit recording, facilitator assignment, log display).

## Implementation Details

- The diff mirrors exactly what the edit persists: only keys present in the `SessionUpdateData` update dict are compared.
- Both edit paths go through `SessionContentEditService.apply()` — a single atomic unit, so a partial edit (e.g. a failing facilitator write) can never leave committed content without its audit row.

## Screenshots

Captured via Playwright against the seeded e2e environment (Polish UI).

**Proposals page header — a new "Dziennik zmian" (Activity Log) action appears:**

| Before | After |
| --- | --- |
| proposals before | proposals after |

**New Content Activity Log view** — Tessera-styled table with per-edit field-level diffs:

content activity log

> Change labels are localized at render time in the active language — core columns via the `content_field_label` filter, dynamic session-field names resolved live from the event. The screenshot shows the Polish UI (`Tytuł`, `Opis`, `Limit uczestników`, `Minimalny wiek`). The log stores only identifiers, never UI text.

> Screenshots live on the ephemeral branch `claude/audit-content-edits-screenshots` (images only, not for merge).

https://claude.ai/code/session_01Y9KJFhwmQXDn6rcR1Pwvxa

## Release Notes

* **New Features**
  * Added session content activity log to track all edits with detailed change history, including timestamps and editor information
  * New Content Activity Log panel page accessible from the proposals section

## Release Notes

* **New Features**
  * Content activity log: Now tracks all changes to session proposals, displaying detailed records of who made edits, what fields were modified, and when each change occurred
  * New "Activity Log" button on the proposals management page provides quick access to view the complete change history for your event

### CODE REVIEW

File: src/ludamus/gates/web/django/chronology/panel/views/proposals.py [hunk 1]
```diff
@@ -256,10 +257,16 @@ def post(self, _request: PanelRequest, slug: str, proposal_id: int) -> HttpRespo
...
 
+        field_values = self._collect_session_field_values(session.pk, current_event.pk)
+        self.request.services.session_content_edit.apply(
+            session_id=proposal_id,
+            event_id=current_event.pk,
+            user_id=self.request.context.current_user_id,
+            update=update_data,
+            field_values=field_values,
+        )
         self._update_facilitators(session.pk, current_event.pk)
```
[the remark this answers is not in the document]
Comment (hasparus) [reply src/ludamus/gates/web/django/chronology/panel/views/proposals.py]: Fixed in 5d77594. Rather than wrapping the view in `uow.atomic()`, I folded the facilitator write into `SessionContentEditService.apply()` so the session update, field values, facilitators, and the audit row all commit in the service's single `transaction.atomic()`. This keeps the "services own transactions; views never start them" rule intact. The write payload is bundled into a `SessionContentEditData` DTO; the panel passes the validated facilitator ids, and the self-edit path leaves facilitators untouched (`None`). Covered by `test_post_assigns_facilitators`.

---
_Generated by [Claude Code](https://claude.ai/code)_
